app/shared/Shared/Services/AzureSearchEmbedService.cs

```diff
@@ -449,8 +449,9 @@ private async Task IndexSectionsAsync(IEnumerable<Section> sections)
         var batch = new IndexDocumentsBatch<SearchDocument>();
         foreach (var section in sections)
         {
-            var embeddings = await openAIClient.GetEmbeddingsAsync(new Azure.AI.OpenAI.EmbeddingsOptions(embeddingModelName, [section.Content.Replace('\r', ' ')]));
-            var embedding = embeddings.Value.Data.FirstOrDefault()?.Embedding.ToArray() ?? [];
+            var embeddingsClient = openAIClient.GetEmbeddingClient(embeddingModelName);
+            var embeddings = await embeddingsClient.GenerateEmbeddingAsync(input: section.Content.Replace('\r', ' '));
+            var embedding = embeddings.Value.ToFloats(); // .Value.Data.FirstOrDefault()?.Embedding.ToArray() ?? [];
```

Comment:
Is the comment at the end of this needed?

Reply:
Oh, that was part of the package upgrade tests, deleted the unused comment.